Tools/docker_run.sh

```diff
@@ -47,14 +47,14 @@ CCACHE_DIR=${HOME}/.ccache
 mkdir -p "${CCACHE_DIR}"
 
 docker run -it --rm -w "${SRC_DIR}" \
+	--user="$(id -u):$(id -g)" \
```

Comment:
can you also add this so it works on ARM based environments?

```suggestion
	--user="$(id -u):$(id -g)" \
	--platform linux/amd64 \
```

Reply:
Or is the right solution to have both amd64 and arm64 environments?